Automattic/wp-calypso
# i18n: Fix translations on the Inbox Screen

#### Proposed Changes

* Adds dynamic getters to the translated properties of `googleWorkspaceCardInformation` and `professionalEmailCardInformation`  objects so that `translate` is called every time they are accessed, to make sure the text is refreshed when translations are loaded.

#### Testing Instructions

1. Change your user language to a non-English language.
2. Visit [https://wordpress.com/inbox/](https://wordpress.com/inbox/) and select a test site that has a domain and no email subscription
3. Confirm that the following content is now translated:
![image](https://user-images.githubusercontent.com/23708351/177545167-60381b22-e144-4da5-9acd-a94107a93a73.png)
